src/TesApi.Web/AzureProxy.cs

```diff
@@ -101,7 +101,7 @@ public AzureProxy(IOptions<BatchAccountOptions> batchAccountOptions, BatchAccoun
             else
             {
                 location = batchAccountInformation.Region;
-                batchClient = BatchClient.Open(new BatchTokenCredentials(batchAccountInformation.BaseUrl, () => GetAzureAccessTokenAsync(CancellationToken.None, "https://batch.core.windows.net/")));
+                batchClient = BatchClient.Open(new BatchTokenCredentials(batchAccountInformation.BaseUrl, () => GetAzureAccessTokenAsync(CancellationToken.None, "https://batch.core.windows.net//.default")));
```

Comment:
This one should only be one slash.

Reply:
Matt pointed out that double slash is the convention for public cloud actually https://github.com/microsoft/CromwellOnAzure/pull/748#discussion_r1495027845

Reply:
That only applies to the public cloud resource manager (control plane). It does not apply to any data-plane system, nor to the resource manager on the sovereign clouds.